packages/utilities/components/max.ts

```diff
@@ -0,0 +1,17 @@
+function calculate(a, b) {
+  return b > a ? b : a;
+}
+
+function max(a?: number | Date | string, b?: number | Date | string) {
+  if (arguments.length === 0) {
+    return max;
+  }
+  if (arguments.length === 1) {
+    return b => {
+      return calculate(a, b);
+    };
+  }
+  return calculate(a, b);
+}
+
+export default max;
```

Comment:
🎨 Can we inline the default exports in general?

Reply:
I think he wanted to go with double export again 🤔